api/nodes/serializers.py

```diff
@@ -260,6 +260,9 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
         'subjects',
     ]
 
+    subjects_related_view = 'nodes:node-subjects'
+    subjects_related_view_kwargs = {'node_id': '<_id>'}
```

Comment:
I don't love this, still thinking of best way to DRY this code up among the views.